src/app/events/README.md

```diff
@@ -0,0 +1,33 @@
+# Events
+
+This directory contains event handlers for Discord events. Files in this folder will automatically 
+be registered to listen to the Discord event specified by `name`.
+
+A full list of available Discord events can be found on the
+[discord.js documentation](https://discord.js.org/#/docs/main/stable/class/Client).
+
+Evant handlers are structured as follows:
+```typescript
+export default class implements DiscordEvent {
+    /*
+     * Name of the Discord event to handle.
+     */
+    name = 'guildMemberAdd';
+
```

Comment:
Don't fully understand the events in Discord API - I would have assumed we should be listening for 'guildRoleAdd' or something similar?